spec/api/api_base_controller_spec.rb

```diff
@@ -0,0 +1,86 @@
+require "spec_helper"
+require "api/api_helper"
+
+include ApiHelper
+
+describe "Base API Controller" do
+
+  before do
+    allow(User).to receive(:is_archivist?).and_return(true)
+  end
+
+  before(:each) do
+    @under_test = Api::V1::BaseController.new
+  end
+
+  describe "batch_errors" do
+
+    describe "with no archivist" do
+      it "should return the 'forbidden' status" do
+        status, _ = @under_test.instance_eval { batch_errors(nil, api_fields) }
```

Comment:
This `instance_eval` means that the test can exercise the private method `batch_errors` in the controller without having to make it public